web/src/components/tasks/anytime-action/tax-clearance-certificate/TaxClearanceSteps.tsx

```diff
@@ -1,44 +1,63 @@
 import { HorizontalStepper } from "@/components/njwds-extended/HorizontalStepper";
-import { TaxClearanceStepOne } from "@/components/tasks/anytime-action/tax-clearance-certificate/TaxClearanceStepOne";
-import { TaxClearanceStepThree } from "@/components/tasks/anytime-action/tax-clearance-certificate/TaxClearanceStepThree";
-import { TaxClearanceStepTwo } from "@/components/tasks/anytime-action/tax-clearance-certificate/TaxClearanceStepTwo";
+import { CheckEligibility } from "@/components/tasks/anytime-action/tax-clearance-certificate/steps/CheckEligibility";
+import { Download } from "@/components/tasks/anytime-action/tax-clearance-certificate/steps/Download";
+import { Requirements } from "@/components/tasks/anytime-action/tax-clearance-certificate/steps/Requirements";
+import { Review } from "@/components/tasks/anytime-action/tax-clearance-certificate/steps/Review";
 import { StepperStep } from "@/lib/types/types";
-import { ReactElement } from "react";
+import { ReactElement, useState } from "react";
 
 interface Props {
   steps: StepperStep[];
+  certificatePdfBlob?: Blob;
   currentStep: number;
   stepIndex: (value: ((prevState: number) => number) | number) => void;
   saveTaxClearanceCertificateData: () => void;
   setStepIndex: (step: number) => void;
-  setCertificatePdfBlob: (
-    value: ((prevState: Blob | undefined) => Blob | undefined) | Blob | undefined
-  ) => void;
 }
 
 export const TaxClearanceSteps = (props: Props): ReactElement => {
+  const [certificatePdfBlob, setCertificatePdfBlob] = useState<Blob | undefined>(
+    props.certificatePdfBlob || undefined
+  );
+
   const onStepClick = (step: number): void => {
     if (step === 2 && props.currentStep === 1) {
       props.saveTaxClearanceCertificateData();
     }
     props.setStepIndex(step);
   };
```

Comment:
Considering simplifying this to just save on every page. That `if` really isn't buying us anything.